### PR TITLE
wip 528

### PR DESCRIPTION
**Pull request by neelghosh**
rangle/angular-training-slides#528

@gvwilson - for review - addressed issue 528 in a11y slides

### CODE REVIEW

File: slides/src/content/a11y.md [hunk 1]
```diff
@@ -173,7 +173,9 @@ For example - error handling where we want to bring the attention of a screen re
 ## Accessible Development - Defining Page Sections
 ###CSS for Navigation and Offscreen Text
 
-- To help navigation for visually impaired users, develop offscreen-text and navigation related CSS classes to mark up page sections in a way hidden to visual users
+- Develop offscreen-text and navigation related CSS classes to mark up page sections in a way hidden to visual users
```
Comment (gvwilson) [inline slides/src/content/a11y.md]: "off-screen text" maybe?

File: slides/src/content/a11y.md [hunk 2]
```diff
@@ -173,7 +173,9 @@ For example - error handling where we want to bring the attention of a screen re
...
+- Off screen text can provide additional valuable information audibly needed by screen reader users for clarity
+- Screen readers re-read everything re-load. 'Skip to nav' allows users to by-pass what they've already heard
```
Comment (gvwilson) [inline slides/src/content/a11y.md]: - "on every re-load" (words missing)
- double quotes around "Skip to nav"

Comment (gvwilson) [reply slides/src/content/a11y.md]: "bypass" not "by-pass"

File: slides/src/content/a11y.md [hunk 3]
```diff
@@ -215,6 +217,36 @@ For example in your header div:
 
 ---
 <!-- .slide: id="a11y" -->
+## Accessible Development - Defining Page Sections
+###CSS Clip Attribute
```
Comment (gvwilson) [inline slides/src/content/a11y.md]: We aren't using sub-headings in other slides - rename this slide (and the ones that follow) to just be "CSS Clip Attribute" maybe?

File: slides/src/content/a11y.md [hunk 4]
```diff
@@ -215,6 +217,36 @@ For example in your header div:
...
+</style>
+</head>
+<body>
+
+<h1 class=".clp">This is offscreen text</h1>
+</body>
+```
+
+---
+<!-- .slide: id="a11y" -->
```
Comment (gvwilson) [inline slides/src/content/a11y.md:244]: Example of what this looks like?

**Review by gvwilson — COMMENTED** [review]
Minor changes.

File: slides/src/content/a11y.md (hunk 4 above)
Comment (neelghosh) [reply slides/src/content/a11y.md:244]: see my comments on issue 530 - I've created a separate skip to nav example.- how it plays out with voice over (VO)

Comment (gvwilson) [reply slides/src/content/a11y.md:244]: thanks - merging.

**Pull request merged**